packages/manager/modules/iam/src/resolves/misc.resolve.js

```diff
@@ -79,6 +79,34 @@ statementResolve.resolves = [entityResolve];
 
 // ---------------------------------------------------------------------------------------------------- //
 
+/**
+ * Get the status of the advanced mode
+ * @returns {boolean}
+ */
+const advancedModeResolve = /* @ngInject */ ($q, PreferencesService) => {
+  return $q
+    .resolve(PreferencesService.getPreference(KEY_PREFERENCES.advancedMode))
+    .then(({ value }) => {
+      return value === 'true';
+    })
+    .catch((err) => {
+      if (err.status === 404) {
+        $q.resolve(
+          PreferencesService.createPreference(
+            KEY_PREFERENCES.advancedMode,
+            'false',
```

Comment:
**question:** Why use `false` as a string?

Reply:
The same of https://github.com/ovh/manager/pull/9146#discussion_r1160733549
the api /me/preferences/manager use a string / text value.